operators/constellation-node-operator/internal/cloud/gcp/client/scalinggroup.go

```diff
@@ -129,7 +131,7 @@ func (c *Client) ListScalingGroups(ctx context.Context, uid string) ([]cspapi.Sc
 			}
 			template, err := c.instanceTemplateAPI.Get(c.projectID, templateURI[len(templateURI)-1])
 			if err != nil {
-				return nil, fmt.Errorf("getting instance template: %w", err)
+				retErr = errors.Join(retErr, fmt.Errorf("getting instance template %q: %w", templateURI[len(templateURI)-1], err))
```

Comment:
```suggestion
				retErr = errors.Join(retErr, fmt.Errorf("getting instance template %q: %w", templateURI[len(templateURI)-1], err))
				continue
```
Need to skip to the next loop iteration in case of errors